exercises/atbash-cipher/example.ml

```diff
@@ -0,0 +1,57 @@
+let substitute = function
```

Comment:
It was an oversight. Implementing take and drop distracts from the actual exercise.

Besides that, seeing that I made an incorrect implementation, I am in favour of *drop*ping (pun intended) the custom implementation for the once in Core